cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala

```diff
@@ -85,14 +100,26 @@ If any other node with that label is updated or created with a `name` that
 already exists, the write operation will fail.
 This constraint will create an accompanying index.
 
+###assertion=create-named-unique-property-constraint
+//
+
+CREATE CONSTRAINT unique ON (p:Person)
+       ASSERT p.age IS UNIQUE
+###
+
+Create a unique property constraint on the label `Person` and property `age` with the name `unique`.
+If any other node with that label is updated or created with a `age` that
+already exists, the write operation will fail.
+This constraint will create an accompanying index.
+
 ###assertion=drop-unique-property-constraint
 //
 
 DROP CONSTRAINT ON (p:Person)
      ASSERT p.name IS UNIQUE
 ###
 
-Drop the unique constraint and index on the label `Person` and property `name`.
+Drop the unique constraint and index on the label `Person` and property `name`. This is deprecated.
```

Comment:
SInce the refcard should be a quick reference, I think we should remove all the entries that are now deprecated from there. It is enough that they are in the Cypher manual.